tests/simulation/shard_rsm.cpp

```diff
@@ -480,6 +480,56 @@ std::tuple<size_t, std::optional<msgs::VertexId>> AttemptToScanAllWithExpression
   }
 }
 
+msgs::GetPropertiesResponse AttemptToGetProperties(ShardClient &client, std::vector<PropertyId> properties,
```

Comment:
I would avoid investing any time in these `shard_rsm` tests since they are cluttered and unorganized, better add simple unit tests in `tests/unit/storage_v3_shard_rsm.cpp`

Reply:
Upon discussing this offline, we decided to not migrate the tests for now